File: Ruby/counting_sheep.rb
def count_sheep(array)
  count = 0
  for s in array
    count += 1 if s == true
  end
  return count
end

test_array = [true, true, true, false,
              true, true, true, true,
              true, false, true, false,
              true, false, false, true,
              true, true, true, true,
              false, false, true, true]

p count_sheep(test_array)

File: Python/counting_sheep.py
def count_sheeps(sheep):
    count = 0
    for b in sheep:
        if b == True:
            count += 1
    return count


count_sheeps([True,  True,  True,  False,
              True,  True,  True,  True,
              True,  False, True,  False,
              True,  False, False, True,
              True,  True,  True,  True,
              False, False, True,  True])
